File: Chapter 03/Java/src/main/java/com/grokkingsoftwarearchitecture/chapter03/section_3_4_order_processor_refactor/before/Order.java
package com.grokkingsoftwarearchitecture.chapter03.section_3_4_order_processor_refactor.before;

import java.util.List;

/**
 * A Simple Data Transfer Object (DTO).
 * * ARCHITECTURE NOTE: This class is actually fine! It contains no logic, just state. 
 * The problem isn't the data; the problem is how the data is processed.
 */
public class Order {
    public List<String> items;
    public double total;
    public String customerEmail;

    public Order(List<String> items, double total, String customerEmail) {
        this.items = items;
        this.total = total;
        this.customerEmail = customerEmail;
    }
}

File: Chapter 03/C#/Section_3_4_OrderProcessor_Refactor/Before/Order.cs
using System.Collections.Generic;

namespace Chapter03.OrderProcessorRefactor.Before;

/// <summary>
/// A Simple Data Transfer Object (DTO).
/// 
/// ARCHITECTURE NOTE: This class is actually fine! It contains no logic, just state. 
/// The problem isn't the data; the problem is how the data is processed.
/// </summary>
public class Order
{
    public required List<string> Items { get; set; }
    public decimal Total { get; set; }
    public required string CustomerEmail { get; set; }
}